data/hai/hai reveal 5 philosophers.txt

```diff
@@ -979,51 +954,44 @@ mission "Hai Rescue [C08] Freedom"
+			`	"There is no evidence that any civilization has ever reached this region of space until us. There are many space-faring indigenous creatures that live in gas giants, and sometimes venture beyond. Most are harmless, so long as you leave them alone. There are flocking apex predators who hunt anything that moves.`
+			`	"The planet below is a mystery to us. It has a thousand times the mass it realistically should, and its gravitational field reads as such from a great distance. This tremendous gravitational field sustains the large asteroid belt. Despite its incredible mass, as you approach the surface, the field does not increase at anywhere near the rate that it should. Thus, when you stand upon it, the gravity is only similar to that of Earth. There are many forms of radiation and strange materials inside the planet's caves. Perhaps, now that we are not developing pirate weapons, we can focus on exploration."`
 				goto questions
 			label leave
-			`	As you return to your ship, Teeneep interrupts you. "I need you to escort me one last time, Captain <last>. I must return to <destination> to resolve things with now that the threat is ended. You will be well-paid, of course."`
-			`	Anticipating your next question, she says, "The ships of the True Hai will be escorted back with us since the Philosophers are convinced they are at no risk, and still have a ship at their disposal. Some of the True Hai have chosen to remain here and join the philosophers. More than enough bunks have subsequently been freed to return those who have chosen to come home with us. We will have to transport the surviving operatives back to Foundry, the Republic should be able to look after them again from there."`
+			`	As you return to your ship, Teeneep interrupts you. "I need you to escort me one last time, Captain <last>. I must return to <destination> to resolve things now that the threat is ended. You will be well-paid, of course."`
+			`	Anticipating your next question, she says, "The ships of the Unfettered Hai will be escorted back with us since the Philosophers are convinced they are at no risk and still have a ship at their disposal. Some of the Unfettered have chosen to remain here and join the philosophers. More than enough bunks have subsequently been freed to return those who have chosen to come home with us. The surviving operatives are planning how to back to Foundry themselves."`
```

Comment:
```suggestion
			`	Anticipating your next question, she says, "The ships of the Unfettered Hai will be escorted back with us since the Philosophers are convinced they are at no risk and still have a ship at their disposal. Some of them have chosen to remain here and join the philosophers. More than enough bunks have subsequently been freed to return those who have chosen to come home with us. The surviving operatives are planning how to back to Foundry themselves."`
```